src/main/java/reposense/authorship/analyzer/AnnotatorAnalyzer.java

```diff
@@ -102,18 +99,23 @@ public static Optional<String> extractAuthorName(String line) {
                 .map(l -> l.split(AUTHOR_TAG))
                 .filter(array -> array.length >= 2)
                 // separates by end-comment format to obtain the author's name at the zeroth index
-                .map(array -> array[1].trim().split(COMMENT_FORMATS[getCommentTypeIndex(line)][1]))
+                .map(array -> COMMENT_FORMATS[getCommentTypeIndex(line)][1] != null
+                        ? array[1].trim().split(COMMENT_FORMATS[getCommentTypeIndex(line)][1])
+                        : array[1].trim().split("", 1))
```

Comment:
Sorry, can I check what is this line for? When you do a `.split("", 1)` you will get back an array of length 1, and the only member of the array is the original string. Is this intended?

If that was intentional, I think we should instead create the String array using the proper syntax instead of using this.

Reply:
Sorry, I missed out your comment in the PR description. Is it possible to change to use `Arrays.copyOfRange()` instead?

Reply:
Apologies, I just tried again with the `new` keyword to create a new String array and it turns out it is possible. I must have missed out a parentheses when trying that out. Will update to use the explicit String[] constructor!